packages/build-tools/src/ios/pod.ts

```diff
@@ -1,12 +1,20 @@
 import path from 'path';
 
+import fs from 'fs-extra';
 import { Ios } from '@expo/eas-build-job';
 import spawn from '@expo/turtle-spawn';
 
 import { BuildContext } from '../context';
 
 export async function installPods<TJob extends Ios.Job>(ctx: BuildContext<TJob>): Promise<void> {
   const iosDir = path.join(ctx.reactNativeProjectDirectory, 'ios');
+
+  if (ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL && Math.random() < 0.33) {
```

Comment:
```suggestion
  if (ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL) {
```
move that to worker code